main.go

```diff
@@ -1,6 +1,9 @@
 package main
 
 import (
+	"fmt"
+	"github.com/pkg/errors"
```

Comment:
I've been putting external packages underneath the other packages, separated by a line of whitespace

Reply:
Both packages were not needed anymore and have been removed.

Reply:
Do you mind running `go mod tidy` to remove the errors package from the `go.mod` and `go.sum` files?

Reply:
Done.